test/integration/IntegrationTest.cpp

```diff
@@ -1338,35 +1341,19 @@ TEST_F(IntegrationTest, audioOnlyNoPadding)
         *_transportFactory,
         *_sslDtls);
 
+    GroupCall<SfuClient<ColibriChannel>> groupCall({&client1, &client2, &client3});
+
     // Audio only for all three participants.
     client1.initiateCall(baseUrl, conf.getId(), true, true, false, false);
     client2.initiateCall(baseUrl, conf.getId(), false, true, false, false);
     client3.initiateCall(baseUrl, conf.getId(), false, true, false, false);
 
-    EXPECT_TRUE(client1._channel.isSuccess());
-    EXPECT_TRUE(client2._channel.isSuccess());
-    EXPECT_TRUE(client3._channel.isSuccess());
-
-    if (!client1._channel.isSuccess() && client2._channel.isSuccess() && client3._channel.isSuccess())
+    if (!groupCall.connect(utils::Time::sec * 8))
     {
+        EXPECT_TRUE(false);
```

Comment:
Makes the intention clear and gives a better message about the failing when test fail because of thos
```suggestion
        FAIL() << "Clients not connected on time";
```